MSAL/src/public/MSALUser.h

```diff
@@ -91,8 +77,9 @@
                 environment:(NSString *)environment;
 
 /*!
-    Returns the unique identifier of the user, which is a combination of uid and utid properties
+ Returns the unique identifier of the user, which is a combination of uid and utid properties
```

Comment:
UserIdentifier is generated based on uid&utid or it uses uniqueIdentifier?

Reply:
it is uid&utid